internal/service/advancedcluster/model_advanced_cluster.go

```diff
@@ -15,7 +15,8 @@ import (
 	"github.com/mongodb/terraform-provider-mongodbatlas/internal/common/constant"
 	"github.com/mongodb/terraform-provider-mongodbatlas/internal/common/conversion"
 	"github.com/spf13/cast"
-	"go.mongodb.org/atlas-sdk/v20231115014/admin"
+	admin20231115 "go.mongodb.org/atlas-sdk/v20231115014/admin"
+	"go.mongodb.org/atlas-sdk/v20240530001/admin"
```

Comment:
preview version will be called with `admin`, while currrent stable version will use alias `admin20231115` as this will become the fixed old sdk version.